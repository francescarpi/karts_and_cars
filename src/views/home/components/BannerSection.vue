<script setup lang="ts">
import LogoKartsAndCars from "../../../components/LogoKartsAndCars.vue";
</script>

<template>
  <div class="h-72 relative">
    <div class="bg absolute inset-0"></div>
    <div class="absolute inset-0 text-center">
      <LogoKartsAndCars class="mt-16 mb-4" />
      <h2 class="font-default text-gray-1000 text-2xl">
        Apasionados del motor<br />a tu servicio
      </h2>
    </div>
  </div>
</template>

<style scoped lang="scss">
.bg {
  @apply bg-no-repeat bg-cover h-full w-full opacity-70;
  background-image: url("@/assets/imgs/cars1.jpeg");
}
</style>
