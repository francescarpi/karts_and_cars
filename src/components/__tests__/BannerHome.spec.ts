import BannerSection from "@/views/home/components/BannerSection.vue";
import { mount } from "@vue/test-utils";
import { describe, it, expect } from "vitest";

describe("BannerSection", () => {
  it("renders properly", () => {
    const wrapper = mount(BannerSection);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
