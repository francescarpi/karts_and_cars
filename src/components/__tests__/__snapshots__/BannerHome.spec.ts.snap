// Vitest Snapshot v1

exports[`BannerSection > renders properly 1`] = `
"<div class=\\"h-72 relative md:h-96\\" data-v-c570277c=\\"\\">
  <div class=\\"bg absolute inset-0\\" data-v-c570277c=\\"\\"></div>
  <div class=\\"absolute inset-0 text-center md:max-w-6xl md:mx-auto\\" data-v-c570277c=\\"\\">
    <h1 class=\\"text-5xl mt-12 mb-4\\" data-v-c570277c=\\"\\"><span class=\\"text-primary font-faster\\">Karts</span><span class=\\"ml-2 mr-4 text-xl text-black\\">&amp;</span><span class=\\"text-secondary font-faster\\">Cars</span></h1>
    <h2 class=\\"font-default text-black text-2xl bg-white opacity-75 m-10 p-4 rounded-xl drop-shadow\\" data-v-c570277c=\\"\\"> Apasionados del motor,<br data-v-c570277c=\\"\\">a tu servicio! </h2>
  </div>
</div>"
`;
